extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,346 @@ limitations under the License.
 
-Elements which are common across multiple pages can be programmatically
-hidden in child documents, to avoid e.g. stacking up multiple page footers at
-the end of the document, or to hide the page header from each subsequent page.
+- Avoiding stacking common page elements, such as footers.
+- Avoiding duplicate page headers in subsequent pages
+- Preventing multiple sidebars, etc.
 
-Elements can be hidden by specifying one or more string CSS selectors in the
-`hideSelectors` key of the element config. Elements matching any of the
-selectors will be set to `display: none` in all child documents.
+Elements with the `next-page-hide` attribute are set to `display: none` when loaded as a suggestion inside `<amp-next-page>`. These elements are visible when the document is loaded as a top-level host.
 
 ```html
-<amp-next-page>
-  <script type="application/json">
-    {
-      "hideSelectors": [
-        ".header",
-        ".main footer",
-        "#navigation"
-      ],
-      "pages": ...
-    }
-  </script>
-</amp-next-page>
+<header class="my-header" next-page-hide>
+  <h2>Text here.</h2>
+</header>
 ```
 
+In some cases, you may want to preserve the last instance of elements with fixed position and multiple instances. Such elements may include sticky ads, sticky footers, or notification banners. In this case, the `next-page-replace` attribute comes in handy. To preserve the last instance, choose a common identifying value for each of these types of elements. Use that common identifying value as the `next-page-replace` attribute's value on each element instance.
+
+In the following example, as the user scrolls down, the first sticky element will get replaced by the second instance of the type.
+
+_on the first document_
+
+```html
+<div class="sticky" next-page-replace="sticky-123">
+  <h2>The second sticky will replace me once you scroll past my page</h2>
+</div>
+```
+
+_on the second document_
+
+```html
+<div class="sticky" next-page-replace="sticky-123">
+  <h2>I replaced the first instance of my type (sticky-123)</h2>
+</div>
+```
+
+### Migrating from 0.1
+
+The experimental `0.1` version of `amp-next-page` had a similar but more restricted API. Version `1.0` allows for an infinite number of suggestions and has advanced features such as templated separators and footers. To make use of these features, follow these instructions:
+
+1. Update your `<script custom-element>` tag to link to the `1.0` bundle of `amp-next-page`
+2. Make sure that `amp-next-page` is the last child of the body element, move any footers or other components that used to follow `<amp-next-page>` inside the `<amp-next-page>` tag.
+3. If you were using an inline configuration, the JSON config is now an `array` of pages instead of an `object` with a `pages` entry. Additionally, the `ampUrl` key of each page entry was renamed to `url`.
```

Comment:
```suggestion
3. If you were using an inline configuration, the JSON config is now an `array` of pages instead of an `object` with a `pages` entry. Additionally, you must rename the `ampUrl` key of each page to `url`. 

```